src/services/resource.js

```diff
@@ -81,10 +99,70 @@ module.exports = class resourceHelper {
 
 				const inReviewResources = await resourceQueries.findAll(filter, ['id'])
```

Comment:
@adithyadinesh0412 in review resource means status will be in_review right, in filter that status will be all right ?

Reply:
it's for submitted for review page. so all those statuses we are considering. 
I'll change the variable name. @priyanka-TL